gen3/bin/kube-setup-secrets.sh

```diff
@@ -227,6 +227,13 @@ if [[ -f "${WORKSPACE}/${vpc_name}/creds.json" ]]; then # update fence secrets
   fi
 fi
 
+if ! g3kubectl get configmaps/isb-cgc-link-list > /dev/null 2>&1; then
+  if [[ ! -f "./apis_configs/isb-cgc-link-list.txt" ]]; then
+    touch "apis_configs/isb-cgc-link-list.txt"
```

Comment:
we should not use their name in our code, need to call this something else. external-google-link-list or something